readme.md

```diff
@@ -606,9 +608,11 @@ Unless either [synchronous methods](#execasyncfile-arguments-options), the value
 - file URL.
 - web [`WritableStream`](https://developer.mozilla.org/en-US/docs/Web/API/WritableStream).
 
+This can be an [array of values](#redirect-stdinstdoutstderr-to-multiple-destinations) such as `['inherit', 'pipe']` or `[filePath, 'pipe']`.
+
 #### stderr
 
-Type: `string | number | stream.Writable | WritableStream | URL`\
+Type: `string | number | stream.Writable | WritableStream | URL | StderrOption[]`\
```

Comment:
`StderrOption` doesn't mean anything to a user. It should either be expanded or linked to the types or something.

Reply:
Could we something like this (to avoid the repetition, which might make the option look more complicated than it is)?

```
Type: `string | number | stream.Writable | WritableStream | URL`, or an `Array` of those types\
```

Reply:
👍

Reply:
Done in 222ebdfe7142afb0979cd8314b021a9588a2e077